test/spec/com/cinergix/mapper/ObjectMapperTestAbstract.java

```diff
@@ -1,7 +1,9 @@
 package com.cinergix.mapper;
 
+import java.lang.reflect.Field;
 import java.sql.ResultSet;
 import java.sql.SQLException;
+import java.util.HashMap;
 
 import org.junit.After;
 import org.junit.AfterClass;
```

Comment:
This class is not particularly necessary. The interface extending method can be used inside the test class.